### ISSUE TITLE
Should library depend on env variables?

### ISSUE THREAD

**Issue by realmallaury** [teltech/logger#28]
https://github.com/teltech/logger/blob/f8b50fab3b26d47d333d90f8ac50cc5c0711ece6/logger.go#L102

This looks like an antipattern, it's much easier if only executable files should depend on env variables and libraries are initialized programmatically and decoupled from underlying os.

**Comment by mnakama** [comment]
I agree with you on this. I think it's worth fixing and setting the module version to v2.
